src/commands/ring/index.ts

```diff
@@ -1,6 +1,6 @@
 import { Command } from "../command";
 
-const subcommands = ["add", "delete", "set-default"];
+const subcommands = ["create", "delete", "set-default"];
```

Comment:
@mtarng does the documentation reflect this name change?

Reply:
to my understanding, all docs (at least the ones @bnookala and I wrote) referenced `create` to begin with -- and the `create.md` for rings mentions `create`

Reply:
Yes. All docs refer to `spk ring create`. I had originally created the command as `spk ring add`, but this corrects it.